### PR TITLE
[Customer] Fix function apply voucher && update cart

### PR DESCRIPTION
**Pull request by DTHNhung**
## Related Tickets:

-   [#46900](https://edu-redmine.sun-asterisk.vn/issues/46900)

## WHAT this PR does?

-   Apply voucher

### CODE REVIEW

**Review by khieutrung12 — APPROVED** [review]

**Comment by DTHNhung** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
conflict

**Comment by DTHNhung** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -52,7 +52,14 @@ public function store(StoreRequest $request)
...
             $code = Str::random(config('app.limitRandomString'));
             $data = Session::get('data');
-            $voucher_id  = $data['voucher']->id;
+
+            if (!isset($data['voucher'])) {
+                $voucher_id  = null;
+            } else {
+                $voucher_id  = $data['voucher']->id;
+                $voucher = Voucher::where('id', $voucher_id)
+                ->update([ 'quantity' => $data['voucher']->quantity - 1]);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/OrderController.php]:
```suggestion
                $voucher = Voucher::where('id', $voucher_id)
                    ->update([ 'quantity' => $data['voucher']->quantity - 1]);
```

**Comment by DTHNhung** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**